samples/README.md

```diff
@@ -29,21 +29,16 @@ View the [source code][video_0_code].
 __Usage:__ `node analyze.js --help`
 
 ```
-analyze.js <command>
-
 Commands:
-  analyze.js faces <gcsUri>        Analyzes faces in a video stored in Google Cloud Storage using the Cloud Video
-                                   Intelligence API.
-  analyze.js shots <gcsUri>        Analyzes shot angles in a video stored in Google Cloud Storage using the Cloud Video
-                                   Intelligence API.
-  analyze.js labels-gcs <gcsUri>   Labels objects in a video stored in Google Cloud Storage using the Cloud Video
-                                   Intelligence API.
-  analyze.js labels-file <gcsUri>  Labels objects in a video stored locally using the Cloud Video Intelligence API.
-  analyze.js safe-search <gcsUri>  Detects explicit content in a video stored in Google Cloud Storage.
+  faces <gcsUri>        Analyzes faces in a video stored in Google Cloud Storage using the Cloud Video Intelligence API.
```

Comment:
I think this is out of date. The other repos use the `{filename} {command} {arguments}` notation. Example: https://github.com/googleapis/nodejs-storage/blob/master/samples/README.md

Reply:
I agree that looks wrong, but I also _just_ generated it yesterday. Will look into it when I have a chance (definitely this week as this goes out Wednesday).

Reply:
I just checked this on-branch and it is correct. If you look above you see the `node analyze.js --help` part. (Note that this is the sample readme and not the main one).